contracts/verification/LivepeerVerifier.sol

```diff
@@ -135,10 +135,10 @@ contract LivepeerVerifier is Manager, IVerifier {
 
         // Check if transcoded data hash returned by solver matches originally submitted transcoded data hash
         if (q.commitHash == _result) {
-            IVerifiable(controller.getContract(keccak256("JobsManager"))).receiveVerification(q.jobId, q.claimId, q.segmentNumber, true);
+            IVerifiable(controller.getContract(keccak256("TicketBroker"))).receiveVerification(q.jobId, q.claimId, q.segmentNumber, true);
```

Comment:
@yondonfu can you please remind me the new verification flow and what changes remain for us to make before these lines of code actually work? (since currently `TicketBroker` does not implement the `IVerifiable` interface)

Reply:
At the moment, there are 2 options for implementing the new verification workflow:

1. Implement the `IVerifiable` interface and the `verify()` function in `LivepeerTicketBroker`

2. Make implementations of the `IVerifier` interface standalone and have a broadcaster directly interact with these contracts instead of going through `LivepeerETHTicketBroker`

I am currently leaning toward 2 for the following reasons:

- Separation of concerns such that `LivepeerETHTicketBroker` doesn't need to know about verification
- Less inter-contract calls. With 1 the contract call flow upon completion of the external computation is external process -> `IVerifier` -> `IVerifiable` -> `BondingManager` (if slashing needs to occur). With 2 the contract call flow is external process -> `IVerifier` -> `BondingManager` (if slashing needs to occur)

Up for discussing the best approach here when we actually get to implementing the new verification workflow

Reply:
I'd love to discuss this further f2f to better understand the solution 2!